test_src/sljitTest.c

```diff
@@ -6281,10 +6282,33 @@ static void test63(void)
 	successful_tests++;
 }
 
+static SLJIT_FUNC sljit_u8 *test64_malloc_exec(sljit_uw size)
```

Comment:
Seems linux specific.

Reply:
yes, since I was assuming you would test it there, eventhought it will build in alll POSIX and also do the right kind of allocation in FreeBSD; indeed it is all described in the description for the PR above